### PR TITLE
Day 04 - Guessing Game.

### PR DESCRIPTION
**Pull request by Phonbopit**
Add Simple Guessing Game.

### CODE REVIEW

**Comment by patmaddox** [comment]
Go you!

File: 2_adventures/001/phonbopit/04_guessing_game/guessing_game.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+secret_number = rand(100) + 1
+
+puts "Welcome to Guessing Game :)"
+puts "-" * 20
+
+won = false
+
+5.times do
+	print "Choose a number between 1 - 100  ===> "
+	guess = gets.chomp.to_i
```
Comment (kotp) [inline 2_adventures/001/phonbopit/04_guessing_game/guessing_game.rb:10]: [Is it ever necessary to use `.chomp` with `.to_i`](http://stackoverflow.com/q/20598679/485864)?

**Comment by kotp** [comment]
LGTM

**Comment by patmaddox** [comment]
nice! I love the guessing game program :)

**Comment by kotp** [comment]
My version is `"What is your favorite number?"  "Really?  Mine is #{your_favorite_number - 1}... I think yours is better!".`

File: 2_adventures/001/phonbopit/04_guessing_game/guessing_game.rb (hunk 1 above)
Comment (Phonbopit) [inline 2_adventures/001/phonbopit/04_guessing_game/guessing_game.rb:10]: @kotp thank for your link :)

**Pull request merged**